src/serac/physics/tests/solid_dynamics_patch.cpp

```diff
@@ -0,0 +1,438 @@
+// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/solid_mechanics.hpp"
+
+#include <functional>
+#include <set>
+#include <string>
+
+#include "axom/slic/core/SimpleLogger.hpp"
+#include <gtest/gtest.h>
+#include "mfem.hpp"
+
+#include "serac/mesh/mesh_utils.hpp"
+#include "serac/physics/state/state_manager.hpp"
+#include "serac/physics/materials/solid_material.hpp"
+#include "serac/serac_config.hpp"
+
+namespace serac {
+
+using solid_mechanics::direct_dynamic_options;
+
+/**
+ * @brief Specify the kinds of boundary condition to apply
+ */
+enum class PatchBoundaryCondition
+{
+  Essential,
+  Mixed_essential_and_natural
```

Comment:
The second enum value name format looks strange to me, it looks like most of serac's `enum class` definitions are using `PascalCase` for their values. It looks like I'm guilty of writing the most enums that don't follow this pattern, so I should go and fix those too!